code/game/jobs/job/medical.dm

```diff
@@ -303,3 +303,33 @@
 	satchel = /obj/item/storage/backpack/satchel_med
 	dufflebag = /obj/item/storage/backpack/duffel/med
 	messengerbag = /obj/item/storage/backpack/messenger/med
+
+/datum/job/officer/med
+	title = "Medical Officer"
+	flag = MEDSEC
+	department = "Medical"
+	department_flag = MEDSCI
+	faction = "Station"
+	total_positions = 1
+	spawn_positions = 1
+	supervisors = "the chief medical officer and head of security"
+	selection_color = "#ed66b1"
+	economic_modifier = 4
+
+	minimum_character_age = 18
+
+	access = list (access_security, access_brig, access_sec_doors, access_eva, access_weapons, access_medical, access_morgue, access_maint_tunnels, access_external_airlocks, access_psychiatrist, access_paramedic)
```

Comment:
Please remove the Paramedic access.
Will lead to officers playing paramedics and the hardsuit.
Paramedic Access should never be handed out because of the hardsuit issue, the Officer has no need for it.

Reply:
This is being fixed by adding additional access requirements on the RIG to prevent officers from having access.